utils/src/test/scala/cromwell/pipeline/utils/AkkaTestSources.scala

```diff
@@ -0,0 +1,24 @@
+package cromwell.pipeline.utils
+
+import akka.actor.ActorSystem
+import akka.stream.ActorMaterializer
+import org.scalatest.{ BeforeAndAfterAll, Suite }
+
+trait AkkaTestSources extends BeforeAndAfterAll {
+  this: Suite =>
+//  lazy val actorSystem = ActorSystem("test")
+  implicit lazy val actorSystem = ActorSystem("test")
+//  implicit lazy val materializer: ActorMaterializer = ActorMaterializer()(actorSystem)
```

Comment:
Remove commented lines

Reply:
Looks like I removed all of this, but have forgotten to commit or add this to index!